src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
You've set the font family to Avenir, Arial, Helvetica, sans-serif. This is good, but remember to use fallback fonts - alternative font-family in case the main one doesn't work. (Refer to the CSS checklist)